.github/labeler.yml

```diff
@@ -29,8 +29,8 @@
 - 'src/core/include/**/*'
 - 'src/frontends/common/**/*'
 - 'src/frontends/onnx/frontend/include/**/*'
-- 'src/frontends/tensorflow/**/*'
-- 'src/frontends/paddle/**/*'
+- 'src/frontends/tensorflow/include**/*'
```

Comment:
maybe 'src/frontends/tensorflow/include/**/*' ?

Reply:
it's the same, but I've fixed it to look more clear